primer.md

```diff
@@ -238,23 +238,54 @@ there are errors during the processing of a CloudEvent, the software
 encountering the error is encouraged to use the normal protocol-level error
 reporting to report them.
 
-## Versioning of Attributes
-
-For certain CloudEvents attributes, the entity or data model referenced by its
-value might change over time. For example, `dataschema` might reference one
-particular version of a schema document. Often these attribute values will then
-distinguish each variant by including some version-specific string as part of
-its value. For example, a version number (`v1`, `v2`), or a date (`2018-01-01`)
-might be used.
-
-The CloudEvents specification does not mandate any particular pattern to be
-used, or even the use of version strings at all. This decision is up to each
-event producer. However, when a version-specific string is included, care should
-be taken whenever its value changes as event consumers might be reliant on the
-existing value and thus a change could be interpreted as a "breaking change".
-Some form of communication between producers and consumers should be established
-to ensure the event consumers know what possible values might be used. In
-general, this is true for all CloudEvents attributes as well.
+## Versioning of CloudEvents
+
+For many CloudEvents, the schema of the data within the event might
+change over time. The CloudEvents specification does mandate any
```

Comment:
did you mean to say "...does not mandate..."?

Reply:
Absolutely - well spotted. Not sure how I fouled up that C&P. Will fix and repush.